### ISSUE TITLE
Latent regularization

### ISSUE THREAD

**Issue by Leminhbinh0209** [Leminhbinh0209/FinetuneVAE-SD#1]
TO-DO: As the denoiser of SD is not fine-tuned in this code, training the VAE alone on new dataset may shift the latent space. There may need to regularize the old the current latent vector with old latent vector in the teach-student: `reg = (self.model.encode(x).saples() - old_vae.encode(x).sample().detach()).pow(2)`. But, I don't see its description in [here](https://huggingface.co/stabilityai/sd-vae-ft-mse-original/blob/main/README.md), and at far as my experiment, there is no need for this.

**Comment by eeyrw** [comment]
Why not finetune the decoder only to keep latent intact?

**Comment by Leminhbinh0209** [comment]
Oh right, that's a simple genius idea. Thank you for a great recommendation!
